testing/trino-product-tests/src/main/java/io/trino/tests/product/TestConfiguredFeatures.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import io.trino.tempto.ProductTest;
+import io.trino.tempto.assertions.QueryAssert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static io.trino.tempto.assertions.QueryAssert.assertThat;
+import static io.trino.tests.product.TestGroups.CONFIGURED_FEATURES;
+import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.sql.JDBCType.VARCHAR;
+
+public class TestConfiguredFeatures
+        extends ProductTest
+{
+    @Inject
+    @Named("databases.presto.configured_connectors")
+    private List<String> configuredConnectors;
+
+    @Inject
+    @Named("databases.presto.configured_password_authenticators")
```

Comment:
This is unused here. I don't think it's possible to extract this from a running Trino server, and the only way of testing this is to actually try to log in using different authentication methods. I'll try to write a test that uses this on a fresh head.

Reply:
I removed this, for now, as such test would rely too much on specific environment config, I'm not sure there's much value in it.